inst/pages/differential_abundance.qmd

```diff
@@ -316,50 +316,67 @@ ancombc2_out$res |>
   kable()
 ```
 
-### MaAsLin2
+### MaAsLin3
 
-Let us next illustrate MaAsLin2 [@Mallick2020]. This method is based on
+Let us next illustrate MaAsLin3 [@Mallick2024]. This method is based on
 generalized linear models and flexible for different study designs
 and covariate structures. For details, check their
-[Biobakery tutorial](https://github.com/biobakery/biobakery/wiki/maaslin2).
+[Biobakery tutorial](https://github.com/biobakery/biobakery/wiki/MaAsLin3).
 
-```{r run-maaslin2, warning=FALSE, results="hide"}
+```{r run-maaslin3, warning=FALSE, results="hide"}
 # Load package
-library(Maaslin2)
+library(maaslin3)
 
-# maaslin expects features as columns and samples as rows
-# for both the abundance table as well as metadata
+# MaAsLin3 takes tse as input data
 
 # We can specify different GLMs/normalizations/transforms.
-# specifying a ref is especially important if you have more than 2 levels
-maaslin2_out <- Maaslin2(
-    input_data = as.data.frame(t(assay(tse))),
-    input_metadata = as.data.frame(colData(tse)),
-    output = "DAA example",
-    transform = "AST",
-    fixed_effects = "patient_status",
-    # you can also fit MLM by specifying random effects
-    # random_effects = c(...),
-    reference = "patient_status,Control",
-    normalization = "TSS",
-    standardize = FALSE,
-    # filtering was previously performed
-    min_prevalence = 0)
+maaslin3_out <- maaslin3(input_data = tse,
+     output = "DAA_example",
+     formula = '~ patient_status',
+     normalization = 'TSS',
+     transform = 'LOG',
+     augment = TRUE,
+     standardize = TRUE,
+     max_significance = 0.1,
+     median_comparison_abundance = TRUE,
+     median_comparison_prevalence = FALSE,
+     max_pngs = 10,
+     cores = 1,
+     save_models = TRUE,
+     verbosity = "WARN")
 ```
 
 Which genera are identified as differentially abundant?
 (leave out "head" to see all).
 
-```{r, maaslin2-res}
-maaslin2_out$results |>
-  filter(qval < 0.05) |>
-  kable()
+```{r, maaslin3-res}
+signif_results <- read.csv('DAA_example/significant_results.tsv',
+sep='\t')
```

Comment:
This does not look good. The indentation is not good

Reply:
Is this file the only format for results? I tried to quickly check, but it seems that the summary table is only available in the file